src/Uno.UI/UI/Xaml/WindowHelper.cs

```diff
@@ -1,22 +1,29 @@
-﻿
+﻿#nullable enable
 
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Media;
 
-namespace Uno.UI.Xaml
+namespace Uno.UI.Xaml;
+
+/// <summary>
+/// An Uno Platform specific <see cref="Windows.UI.Xaml.Window"/> helper.
+/// </summary>
+public static class WindowHelper
 {
+
 	/// <summary>
-	/// An Uno Platform specific <see cref="Windows.UI.Xaml.Window"/> helper.
+	/// Sets the Window background
 	/// </summary>
-	public static class WindowHelper
+	public static void SetBackground(this Window window, Brush background)
 	{
-
-		/// <summary>
-		/// Sets the Window background
-		/// </summary>
-		public static void SetBackground(this Window window, Brush background)
-		{
-			window.Background = background;
-		}
+		window.Background = background;
 	}
+
+	/// <summary>
+	/// Retrieves the native Window object backing the given window.
+	/// Only applicable on some targets.
+	/// </summary>
+	/// <param name="window">Window.</param>
+	/// <returns>Native window type or null.</returns>
+	public static object GetNativeWindow(this Window window) => window.NativeWindow;
```

Comment:
From an Uno app, how would I go about getting the `Microsoft.UI.Xaml.Window` instance from the context of a specific `UIElement` instance?

Reply:
I will probably be adding some Uno-specific API for this, but that will be relevant for the multi window feature. However, in many cases the `XamlRoot` will be enough